### ISSUE TITLE
Invalid Macaroon Password

### ISSUE THREAD

**Issue by almostheaven1863** [lightninglabs/lightning-terminal#545]
<img width="1024" alt="Screenshot 2023-05-10 at 2 25 29 PM" src="https://github.com/lightninglabs/lightning-terminal/assets/98767163/19e59821-8fa3-4352-8879-38e54ef4a248">
When I attempt to install Lightning Terminal, I get this error when trying to run it using command litd --uipassword=********

```
satoshi@nodebox:~/downloads/lightning-terminal-linux-amd64-v0.9.2-alpha$ litd --uipassword=********
2023-05-10 19:20:04.170 [INF] LITD: LiT version: 0.9.2-alpha commit=v0.9.2-alpha
2023-05-10 19:20:04.178 [INF] SESS: Checking for schema update: latest_version=0, db_version=0
2023-05-10 19:20:04.178 [INF] LITD: Dialing lnd gRPC server at localhost:10009
2023-05-10 19:20:04.179 [INF] LITD: Listening for http_tls on: 127.0.0.1:8443
----------------------------------------------------------
 Lightning Terminal (LiT) by Lightning Labs               
                                                          
 LND Operating mode      remote                               
 LND Node status         online                               
 LND Alias               ⚡️MontaniSemperLiberi⚡️                               
 LND Version             0.16.2-beta commit=v0.16.2-beta                               
 LiT Version             0.9.2-alpha commit=v0.9.2-alpha                               
 Web interface           127.0.0.1:8443 (open https://127.0.0.1:8443 in your browser)  			           
----------------------------------------------------------
2023-05-10 19:20:04.188 [INF] LITD: Connecting basic lnd client
2023-05-10 19:20:04.389 [INF] LITD: Connecting full lnd client
2023-05-10 19:20:04.389 [INF] LNDC: Creating lnd connection to localhost:10009
2023-05-10 19:20:04.389 [INF] LNDC: Connected to lnd
2023-05-10 19:20:04.393 [INF] LNDC: Waiting for lnd to unlock
2023-05-10 19:20:04.393 [INF] LNDC: Wallet state of lnd is now: Lnd main server is ready for requests
2023-05-10 19:20:04.404 [INF] LNDC: lnd version: v0.16.2-beta, build tags 'autopilotrpc,signrpc,walletrpc,chainrpc,invoicesrpc,watchtowerrpc,neutrinorpc,monitoring,peersrpc,kvdb_postgres,kvdb_etcd,kvdb_sqlite'
2023-05-10 19:20:04.404 [INF] LNDC: Using network mainnet
2023-05-10 19:20:04.404 [INF] LNDC: Waiting for lnd to be fully synced to its chain backend, this might take a while
2023-05-10 19:20:04.414 [INF] LNDC: lnd is now fully synced to its chain backend
2023-05-10 19:20:04.414 [INF] LITD: Starting integrated faraday daemon
2023-05-10 19:20:04.470 [INF] LNDC: Macaroon DB could not be unlocked with the derived shared key. Attempting to unlock with empty password instead
2023-05-10 19:20:04.520 [ERR] LITD: Could not start subservers: error starting macaroon service: unable to unlock macaroon DB: invalid password
error starting macaroon service: unable to unlock macaroon DB: invalid password
```

**Comment by ellemouton** [comment]
Hi @almostheaven1863  - any chance that you started Lit with some older loop/pool/faraday/litd data lying around from a previous installation from another node?

**Comment by almostheaven1863** [comment]
Not from another node, but it had been installed previously months ago. Initially worked. Haven’t tried to use it in a while. 
Is there a way to uninstall all info associated with it and start over?
I tried downloading the latest versions of everything, but I still get the macaroon error when trying to start it. LND is working.

**Comment by guggero** [comment]
You need to remove any folders from your previous installation that could have old data: `~/.lit`, `~/.pool`, `~/.loop` or `~/.faraday`.

**Comment by almostheaven1863** [comment]
is there a quick way to do this or do I just have to manually search for all the locations of all these folders??

**Comment by almostheaven1863** [comment]
I got farther than before. I manually erased all .lit, .pool, .loop, and .faraday. I reinstalled lightning terminal. when I ran it now it tries to start up. I get this error now - 
2023-05-11 13:36:20.705 [ERR] LNDC: Could not receive from interceptor stream: rpc error: code = Unknown desc = RPC middleware not enabled in config
2023-05-11 13:36:20.706 [ERR] LITD: Could not start subservers: context canceled
2023-05-11 13:36:20.706 [INF] LOOPD: Stopping gRPC server
2023-05-11 13:36:20.706 [INF] LOOPD: Stopping REST server
2023-05-11 13:36:20.706 [ERR] RPCS: Unable to receive block notification: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-05-11 13:36:20.706 [INF] LOOPD: Liquidity manager stopped
2023-05-11 13:36:20.706 [INF] LOOP: Swap client terminating
2023-05-11 13:36:20.706 [INF] LOOP: Swap client terminated
2023-05-11 13:36:20.706 [INF] LOOPD: Swap client stopped
2023-05-11 13:36:20.706 [ERR] ACCT: Error in invoice subscription: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-05-11 13:36:20.706 [INF] LOOPD: Daemon exited
2023-05-11 13:36:20.706 [INF] POOL: Received shutdown signal, stopping server
2023-05-11 13:36:20.706 [INF] AUCT: Shutting down auctioneer client
2023-05-11 13:36:20.706 [INF] RPCS: Trader server stopping
2023-05-11 13:36:20.706 [INF] AUCT: Shutting down auctioneer client
2023-05-11 13:36:20.706 [ERR] ACCT: Unable to receive block notification: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-05-11 13:36:20.706 [ERR] RPCS: Unable to receive block notification: channel acceptor receive failed: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-05-11 13:36:20.706 [INF] FNDG: Funding manager stopping
2023-05-11 13:36:20.707 [INF] FNDG: Stopped funding manager
2023-05-11 13:36:20.707 [INF] RPCS: Stopped trader server
2023-05-11 13:36:20.707 [ERR] LNDC: Error closing lnd connection: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-05-11 13:36:20.707 [ERR] LNDC: Error closing lnd connection: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-05-11 13:36:20.707 [ERR] LITD: Error while stopping litd: rpc error: code = Canceled desc = grpc: the client connection is closing
2023-05-11 13:36:20.707 [ERR] LITD: Error shutting down: rpc error: code = Canceled desc = grpc: the client connection is closing
context canceled
